### PR TITLE
Xiao Create Shadow Consistency for the Dashboard

### PR DESCRIPTION
**Pull request by AuroraLovesWhales**
# Description
(PRIORITY LOW) Jae: Create Shadow Consistency for the Dashboard
a. Please add the shadow shown below to the rest of the buttons on that page: 
image
b. Definitely these two areas, but any other areas on this page you think they would look good too: 
image

## Main changes explained:
Added style={{ boxShadow: '2px 2px 4px 1px lightgray', }} to "Add Intangible Time Entry" button, "Badge Report" button, and "CountDownTimer" box. Also added some padding between the three icons next to "Tasks and Timelogs" to make it look better.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log in as any user and verify that the changes have been made

## Screenshots or videos of changes:
After:
image
image
image

## Note:
This is an easy PR to test. The only changes are made to the UI. If you have any other suggestions for improvement, please let me know.

### CODE REVIEW

**Comment by one-community** [comment]
Looks good!

File: package-lock.json
Comment (Cirakuze) [inline package-lock.json:1]: What is this diff from? Is it relevant to the feature changes of this PR?

**Review by Cirakuze — CHANGES_REQUESTED** [review]
This PR does not include any changes to the packages, but includes a package-lock.json diff. Unless the package-lock.json diff is intentional and this PR is missing the accompanying package version diffs, please remove it from this PR.

The three boxShadow styles in this PR seem to be identical. If the intention is for them to always be the same, please explore ways to allow this style to be reusable and scalable, allowing a dev in the future to touch just one part of the code to apply the changes to everywhere this specific style is used.

**Comment by one-community** [comment]
> The three boxShadow styles in this PR seem to be identical. If the intention is for them to always be the same, please explore ways to allow this style to be reusable and scalable, allowing a dev in the future to touch just one part of the code to apply the changes to everywhere this specific style is used.

Great suggestion! Let me know how much time to add for doing this Xiao, setting it up so everything throughout the app with  these shadows can be adjusted with one code change. Then look at all the other pages and see where this can be applied there too... include all the Admin-level pages. So this is a pretty big task because it will touch all pages.

**Comment by AuroraLovesWhales** [comment]
> > The three boxShadow styles in this PR seem to be identical. If the intention is for them to always be the same, please explore ways to allow this style to be reusable and scalable, allowing a dev in the future to touch just one part of the code to apply the changes to everywhere this specific style is used.
> 
> Great suggestion! Let me know how much time to add for doing this Xiao, setting it up so everything throughout the app with these shadows can be adjusted with one code change. Then look at all the other pages and see where this can be applied there too... include all the Admin-level pages. So this is a pretty big task because it will touch all pages.

Thank you for the advice! I will apply these suggestions. I will probably need 3 more hours to do it.

**Review by 20chen-7 — COMMENTED** [review]
Looks great! The shadow is like intended. I agree with the [Cirakuze](https://github.com/Cirakuze), there are three duplicating style.
Screenshot 2023-05-30 at 11 02 23 AM
Screenshot 2023-05-30 at 11 02 19 AM

**Review by NidaZaki — COMMENTED** [review]
This PR works as intended, and I agree with @Cirakuze
PR#862

**Comment by AuroraLovesWhales** [comment]
I have manually deleted package-lock.json from github, but now all checks have failed. Does anyone know how to resolve this? Thank you!

I have added a styles.js file to make the boxShadow style reusable. Here are other places throughout the app I have applied the style too. Please let me know if there are any suggestions or areas for improvement. 

Weekly Summaries Reports
Pasted Graphic 3

Badge Report Popup
Pasted Graphic 4
Pasted Graphic 12

Badge Management
Pasted Graphic 5
Pasted Graphic 6

Profile
Pasted Graphic 13

Weekly Summaries
Pasted Graphic 8

Permission Management
Pasted Graphic 10

WBSTasks
Pasted Graphic 11

**Comment by one-community** [comment]
> I will probably need 3 more hours to do it.

Excellent, added 4 hours to where you are now, to include any other requests here.

File: src/components/Timelog/Timelog.jsx [hunk 1]
```diff
@@ -473,8 +476,8 @@ const Timelog = props => {
                       {isOwner ? (
                         <div className="float-right">
                           <div>
-                            <Button color="success" onClick={toggle}>
-                              {'Add Intangible Time Entry '}
+                            <Button color="success" onClick={toggle} style={boxStyle}>
+                              Add Intangible Time Entry{' '}
```
Comment (lchua2314) [inline src/components/Timelog/Timelog.jsx:480]:
```suggestion
                              Add Intangible Time Entry
```
This is probably the caused by the linter.

**Review by lchua2314 — CHANGES_REQUESTED** [review]
Hi @AuroraLovesWhales I've tried to solve the package-lock.json issue, but I couldn't find a solution. I looked online and people say to not ever delete package-lock.json, but to instead fix conflicts with package.json then run "npm install". Not sure how much that can help you, but I felt like I should inform you of that. 

Also, I requested a small change of some code. It's probably due to the linter.

I tested the changes and everything works as intended, however, I cannot approve your PR until the package-lock.json and my requested change gets fixed.

Weekly Summaries Reports
1 Weekly Summaries Reports

Badge Report Popup
2 Badge Report Popup
3 Badge Report Popup

Badge Management
4 Badge Management
5 Badge Management

Profile
6 Profile

Weekly Summaries
7 Weekly Summaries

WBSTasks
8 WBSTasks

**Review by OAkinbode — COMMENTED** [review]
Hi, 

Nice job. I think the solution is pretty elegant. 

PR 862 working fine

**Comment by AuroraLovesWhales** [comment]
> Hello @AuroraLovesWhales, I also once committed changes in package.json file and deleting it manually, the checks failed. So I pushed again my previous changes, this time, excluding package.json file from my commit. Or you can read this blog from medium https://blog.devgenius.io/how-to-remove-modified-or-changed-files-from-a-git-pull-request-a7769e3c58da.
> 
> Those shadows looks great, I am not sure but are you also planning to do this on reports -> reports -> choose people -> peoplereport page there are two button on below. and also a buttons in bottom of the view profile page. lastly on other links -> popup management page buttons

Hi @papiasharmin. Thank you so much for your feedback! I have raised another PR because I am still unable to solve the issue, and I have taken your advice to add the style to the three areas you pointed out. Please refer to [PR864](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/pull/864) to see the changes. Thank you again!

**Review by OAkinbode — COMMENTED** [review]
Just for information. This is how the shadows look In dark mode (from PR 842). I added the styling to one of the larger components in that PR and I think they look good as well. 

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/104801652/b3498abb-125b-4649-8706-d786f870c75d

**Review by yufeiStella — COMMENTED** [review]
It works well now. Great job!
Screen Shot 2023-06-03 at 02 56 28
Screen Shot 2023-06-03 at 02 55 53

**Pull request closed without merging**